scripts/brain_server.py

```
@@ -0,0 +1,98 @@
+        try:
+            result = subprocess.run(cmd, shell=True, capture_output=True, text=True, cwd=str(pr))
+            return jsonify({"results": result.stdout.splitlines()})
+        except Exception as e:
+            return jsonify({"error": str(e)}), 500
+
+    @app.route("/api/impact/<func_name>")
+    def impact(func_name):
+        pr = app.config['PROJECT_ROOT']
+        cmd = f"tldr impact {func_name} ."
+        try:
+            result = subprocess.run(cmd, shell=True, capture_output=True, text=True, cwd=str(pr))
```

Ruff (0.14.10)

`subprocess` call with `shell=True` identified, security issue

(S602)